src/features/memo/functions/memoOperations.js

```diff
@@ -0,0 +1,37 @@
+import {
+  getLocalStorage,
+  setLocalStorage,
+} from '../../../functions/localStorage';
+
+const MEMO_LOCAL_STORAGE_KEY = 'memo_key';
+
+export const initMemo = () => {
+  setLocalStorage(MEMO_LOCAL_STORAGE_KEY, []);
```

Comment:
```suggestion
  setLocalStorage(MEMO_LOCAL_STORAGE_KEY, []) || [];
```

のように必ず配列が返るように担保してもいいかもしれません！
意図せず null になることを防ぎ予測性を担保するような目的です！

Reply:
initMemo関数は何も返しておらず、もしかするとgetAllMemos関数のreturn部分のことでしょうか？
`return getLocalStorage(MEMO_LOCAL_STORAGE_KEY)`としている部分で、localStorageに値がない場合にnullを返していたのを、ご指摘いただいたように空配列を返すように修正しました。

謝って解釈しておりましたら申し訳ございません、ご指摘いただけますと幸いです 🙇 🙇

Reply:
ああごめんなさい！コメントするとこが間違っていました 🙏 
解釈合っています ☺️